fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/AgilityContext.java

```diff
@@ -178,6 +179,10 @@ default <T> T getPropertyValue(@Nonnull RecordLayerPropertyKey<T> propertyKey) {
      */
     void setCommitCheck(Function<FDBRecordContext, CompletableFuture<Void>> commitCheck);
 
+    default void commit(@Nonnull FDBRecordContext context) {
+        LuceneConcurrency.asyncToSync(FDBStoreTimer.Waits.WAIT_COMMIT, context.commitAsync(), context);
+    }
+
```

Comment:
After this PR, at some point....
I _feel_ like instead of calling `asyncToSync` `AgilityContext` should just have asynchronous methods, and callers can use `asyncToSync`.